Nodejs/Setup/setup.proj

```diff
@@ -19,6 +19,7 @@
     <ProjectFile Include="NodejsTools\NodejsTools.wixproj"/>
     <ProjectFile Include="NodejsToolsInstaller\NodejsToolsInstaller.wixproj"/>
     <ProjectFile Include="..\..\Internal\Nodejs\VsLoggerSetup\VsLogger.wixproj" Condition="'$(IncludeVsLogger)' == 'True'"/>
+    <ProjectFile Include="InstallerTelemetryAction\InstallerTelemetryAction.csproj" Condition="'$(Configuration)' == 'Release'" />
```

Comment:
You made this conditional for release configurations, not actual releases. Essentially we only want to include telemetry when we run the BuildRelease script with the ```-release``` flag.